flow/Stats.h

```diff
@@ -91,14 +91,28 @@ struct Counter : ICounter, NonCopyable {
 
 	Value getIntervalDelta() const { return interval_delta; }
 	Value getValue() const { return interval_start_value + interval_delta; }
-	double getRate() const;				// dValue / dt
-	double getRoughness() const;			// value deltas come in "clumps" of this many ( 1 = periodic, 2 = poisson, 10 = periodic clumps of 10 (nearly) simultaneous delta )
+
+	// dValue / dt
+	double getRate() const;
+
+	// Measures the clumpiness or dispersion of the counter.
+	// Computed as a normalized variance of the time between each incrementation of the value.
+	// A delta of N is treated as N distinct increments, with N-1 increments having time span 0.
+	// Normalization is performed by dividing each time sample by the mean time before taking variance.
+	//
+	// roughness = Variance(t/mean(T)) for time samples t in T
+	//
+	// A uniformly periodic counter will have roughness of 0
+	// A uniformly periodic counter that increases in clumps of N will have roughness of N-1
```

Comment:
This is the one property that I thought was maybe better before (i.e. that clumps of N have roughness N rather than N-1). However, I'm not sure it's really worth it when it makes it more awkward to describe what roughness actually is. In the proposed implementation, it's probably more fair now to call it a normalized variance.